scenarioexec/stepCheckAccounts.go

```diff
@@ -99,6 +99,16 @@ func (ae *VMTestExecutor) checkAccounts(baseErrMsg string, checkAccounts *mj.Che
 					er.CodeHint))
 		}
 
+		if !expectedAcct.CodeMetadata.IsUnspecified() && !expectedAcct.CodeMetadata.Check(matchingAcct.CodeMetadata) {
+			return fmt.Errorf("%s bad account codeMetadata. Account: %s. Want: %s. Have: \"%s\"",
+				baseErrMsg,
+				expectedAcct.Address.Original,
+				oj.JSONString(expectedAcct.CodeMetadata.Original),
+				ae.exprReconstructor.Reconstruct(
+					matchingAcct.CodeMetadata,
+					er.CodeHint)) // TODO: Metadata hint?
```

Comment:
How to approach Metadata hint?

Reply:
Yes, it's appropriate to have a codeMetadata hint
It should always decode as hex.

Reply:
Also NoHint works for now